src/main/java/no/nav/familie/integrasjoner/client/rest/PdlRestClient.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.integrasjoner.client.rest
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.http.sts.StsRestClient
+import no.nav.familie.http.util.UriUtil
+import no.nav.familie.integrasjoner.felles.OppslagException
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsDato
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsdatoRequest
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsdatoResponse
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlRequestVariable
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestOperations
+import java.net.URI
+
+@Service
+class PdlRestClient(@Value("\${PDL_URL}") pdlBaseUrl: URI,
+                    @Qualifier("sts") val restTemplate: RestOperations,
```

Comment:
Denne qualifieren legger til Authorization: Bearer ...... Ser at det er en egen Nav-Consumer-Token header for token i denne. Hvis ikke Authorization brukes, så burde man kanskje lagd en egen restTemplateConfig i ApplicationConfig

Reply:
Pdl krever begge, både Authorization og Nav-Consumer-Token. Sistnevnte er vel litt mindre vanlig (?), så da la jeg til denne headeren eksplisitt i klient-klassen for pdl. Dersom kombinasjonen brukes også andre steder så er jeg enig i at den bør ut i en egen config.